R/prepare_pstr_company.R

```diff
@@ -4,30 +4,18 @@
 #' @param eco_activities A dataframe like [ecoinvent_activities]
 #' @param pstr_prod A dataframe like [pstr_product]
 #' @param comp A dataframe like [ep_companies]
+#' @param isic_tilt_map A dataframe like [isic_tilt_mapper]
 #' @param pstr_comp A dataframe like the corresponding output at company level
 #'   from tiltIndicator.
 #'
 #' @return A dataframe that prepares the final output of pstr_company
 #'
 #' @export
-#'
-#' @examples
-#' library(tiltIndicator)
-#'
-#' company <- unnest_company(toy_sector_profile_upstream_output())
-#' product <- unnest_product(toy_sector_profile_upstream_output())
-#'
-#' prepare_pstr_company(
-#'   company |> head(3),
-#'   product |> head(3),
-#'   ep_companies |> head(3),
-#'   ecoinvent_activities |> head(3),
-#'   matches_mapper |> head(3)
-#' )
-prepare_pstr_company <- function(pstr_comp, pstr_prod, comp, eco_activities, match_mapper) {
```

Comment:
See my comment above. You don't need to remove the example -- instead hack `toy_sector_profile_upstream_output()` (which lives in this repo) and add a `# FIXME` pointing to an issue or PR in tiltToyData.